### ISSUE TITLE
OSX 10.11.1 - Stuck at "Searching for Sonos devices on network..." 

### ISSUE THREAD

**Issue by MikeTodd** [stephen/airsonos#242]
First off, thanks so much for writing this utility. I just got a two-pack of the PLAY:1s the other day, and not having AirPlay support is my single biggest gripe with Sonos.

I installed node.js via brew, and that installed version 5.1.0 so I'm not sure if that's the problem or not.

When I did "npm install airsonos" I got lots of warnings, but seemed like minor things such as unused variables and comparing integers of different signs. Stuff that shouldn't really break anything.

However, when I run airsonos --diagnostics, I get stuck here:

AirSonos Diagnostics
node version	 v5.1.0
operating sys	 darwin (x64)
ip address	 192.168.0.115

Searching for Sonos devices on network...

Never gets past that. I am using Sonos Controller 6.0 on my Mac. I've tried with and without that open, and it seems to make no difference. But others have mentioned Sonos 6.0 in other threads, so maybe that could be part of the problem too.

Any ideas on what I should try next?

**Comment by tobiastornros** [comment]
I have Sonos Controller 6.0 on my Mac with OS X 10.11.1 and Airsonos works. I think that the problem for you is your Node.js version, 5.1.0 is the latest "stable" version av Node.js. Try downgrading to 4.2.2 (LTS-version). 

Downgrading and installing specific version with Brew: http://apple.stackexchange.com/questions/171530/how-do-i-downgrade-node-or-install-a-specific-previous-version-using-homebrew


Read more about the difference between LTS and latest versions here: https://nodejs.org/en/blog/community/node-v5/

**Comment by MikeTodd** [comment]
Ok, thanks for the advice! So I tried "brew unlink node", then "brew install homebrew/versions/node4-lts". AirSonos gave an error, so I uninstalled and reinstalled that too, and fired it up again. I get the same behavior (but different version of node listed):

$ airsonos --diagnostics
AirSonos Diagnostics
node version	 v4.2.2
operating sys	 darwin (x64)
ip address	 192.168.0.115

Searching for Sonos devices on network...

And then it gets stuck there.
